joomla/joomla-cms
# Restrict choicesjs input on container

Pull Request for Issue #40096.

### Summary of Changes
Every choicesJS input has a min width attribute depending on its placeholder characters. This can be overwritten by just setting the max width of the parent to 100%


### Testing Instructions
Open a page with a choicesJS input (for example the edit article page) and shrink the viewport, the input should adapt


### Actual result BEFORE applying this Pull Request
The input moved out of the viewport


### Expected result AFTER applying this Pull Request
The input stays in the viewport


### Link to documentations
Please select:
